perfkitbenchmarker/providers/ibmcloud/util.py

```diff
@@ -0,0 +1,212 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utilities for working with IBM Cloud resources."""
+
+import os
+import json
+import yaml
+
+from perfkitbenchmarker import errors
+from perfkitbenchmarker.providers.ibmcloud import ibmcloud
+
+WINDOWS = 'windows'  # all windows
```

Comment:
You should be able to use os_types.py for these except for centos (which is centos7 and centos8 in that file)

Reply:
I used some that exists in os_types, it doesn't have ubuntu either, so I left those.